src/screens/Ticketing/FareProducts/RecentFareProducts/RecentFareContracts.tsx

```diff
@@ -94,18 +94,18 @@ export const RecentTickets = () => {
             style={styles.horizontalScrollView}
             testID="recentTicketsScrollView"
           >
-            {memoizedRecentTickets.map((ticket) => {
+            {memoizedRecentFareContracts.map((rfc) => {
               const componentKey =
-                ticket.preassignedFareProduct.id +
-                ticket.userProfilesWithCount
+                rfc.preassignedFareProduct.id +
+                rfc.userProfilesWithCount
                   .map((traveller) => {
                     return traveller.count + traveller.userTypeString;
                   })
                   .join();
               return (
-                <RecentTicketComponent
+                <RecentFareContractComponent
```

Comment:
Think we dont really need to append **component** at the end.

Reply:
Hmm, this kinda triggered an interesting question. We have a type named `RecentFareContract`, and if renaming this we also have a component named `RecentFareContract`. They are in naming and casing exactly the same, and will be hard to distinguish in the code. I think we need an established convention here, so until we have it, I'll just keep the `RecentFareContractComponent` name.

Reply:
Ohh okie,  I would at least like to rename the type in such cases and keep the Component as it is. But lets leave as is for now 👍